src/tests/ftest/control/dmg_pool_query_test.py

```diff
@@ -68,8 +68,8 @@ def test_pool_query_basic(self):
                 "total_targets", path="/run/exp_vals/*"),
             "active_targets": self.params.get(
                 "active_targets", path="/run/exp_vals/*"),
-            "total_nodes": self.params.get(
-                "total_nodes", path="/run/exp_vals/*"),
+            "total_engines": self.params.get(
+                "total_engines", path="/run/exp_vals/*"),
```

Comment:
We need to also update the config:
https://github.com/daos-stack/daos/blob/38ec717e8380758dce844ae905719c225f8884d6/src/tests/ftest/control/dmg_pool_query_test.yaml#L35

Reply:
done